policy/lib/tekton/pipeline.rego

```diff
@@ -35,6 +35,11 @@ pipeline_label_selector(pipeline, selector) := value if {
 	label == selector
 }
 
+pipeline_label_selector(pipeline, selector) := value if {
```

Comment:
I noticed that this file is only tested indirectly. I think we should have a `pipeline_test.rego` file that tests all the rules in this file that don't start with `_`. Could we create one as part of this PR to just cover `pipeline_label_selector`?